### PR TITLE
computed decorate with setter

### PR DESCRIPTION
**Pull request by turboMaCk**
computed decorate with support for setters.

### CODE REVIEW

**Comment by turboMaCk** [comment]
@zdenal ready!

**Comment by turboMaCk** [comment]
closes: https://github.com/GlobalWebIndex/ember-clothier/issues/29

**Comment by turboMaCk** [comment]
this is how new api looks like: https://github.com/turboMaCk/ember-clothier/blob/f52a98a13c35cf1774b08a9c054460da09b8d5cc/README.md#computed-decorate-with-setter

File: README.md [hunk 1]
```diff
@@ -145,14 +145,15 @@ export default Ember.Route.extend({
 ```javascript
 // view/application.js
 import Ember from 'ember';
-import DecorateMixin, { computedDecorate } from 'ember-clothier/decorate-mixin';
+import { computedDecorateWithSetter } from 'ember-clothier/decorate-mixin';
 
-export default Ember.View.extend(DecorateMixin, {
-  activatables: computedDecorate('model', 'activatable'),
+export default Ember.View.extend({
+  model: computedDecorateWithSetter('activatable'),
```
Comment (zdenal) [inline README.md]: Maybe `decoratorFacotry` as we don't decorating (`decorate`) something in this step and doing it later depends on code/user decision.

**Pull request merged**